### DISCUSSION TITLE
Error when running kedro mlflow ui

### DISCUSSION THREAD

**Discussion by snehakumari321** [Galileo-Galilei/kedro-mlflow#423] [Q&A] [answered]
I am following the documentation to use kedro-mlflow with my pipeline registry  and it gives me the following error when running kedro mlflow ui: Any inputs are helpful. Thanks!
`/opt/anaconda3/envs/frontline/lib/python3.8/site-packages/kedro_mlflow/framework/cli/cli.py:161  │
│ in ui                                                                                             
│   158 │   ) as session:   
│   159 │   │   
│   160 │   │   context = session.load_context()                                                   
│ ❱ 161 │   │   host = host or context.mlflow.ui.host 
│   162 │   │   port = port or context.mlflow.ui.port   
│   163 │   │      
│   164 │   │   if context.mlflow.server.mlflow_tracking_uri.startswith("http"):                   │
╰──────────────────────────────────────────────────────────────────────────────────────────────────╯
AttributeError: 'KedroContext' object has no attribute 'mlflow'`

Python version: 3.8.16
Kedro: 0.18.6
kedro-mlflow: 0.11.8

**Comment by Galileo-Galilei** [comment, marked as answer]
Hi @snehakumari321, I am sorry you don't have a good experience with the plugin. 

I've just tested the following code : 

```bash
conda create -n discussion-423 python=3.8.16 -y
conda activate discussion-423
pip install kedro==0.18.6 kedro-mlflow==0.11.8
kedro new --starter=pandas-iris --checkout=0.18.6
cd iris
kedro run 
kedro mlflow ui
```
and the UI opens correctly so I think you have some custom code somewhere because the default seems to work.   

The ``context.mlflow`` attribute is created on the fly when the ``after_context_created`` hook is called by the ``load_context()`` method. It seems it is not called for some unknown reason. Can you answer the following question to help debugging?: 
- did you modify something in your ``settings.py``, e.g. manually registering some hooks?
- does ``kedro run`` works? If yes, are there mlflow related logs? which shows you that something is logged?
- what is your mlflow setup? a local mlruns folder? A remote one?
- Can you show me your ``mlflow.yml`` if you have one?
- Does kedro-viz work? Are there any other plugin installed in your project, including custom ones?  Can you list them? 

Hope I'll be able to help!
